### ISSUE TITLE
Displays nothing...

### ISSUE THREAD

**Issue by cowboysdude** [cbrooker/MMM-Todoist#2]
It literally will not display the list... no errors to go by either.

**Comment by MariussuiraM** [comment]
I have the same problem as cowboysdude

**Comment by cbrooker** [comment]
Thanks for the feedback. I'll take a look at it.

**Comment by svonbentzel** [comment]
i have figured out why the list is blank:

in MMM-Todoist.js is the default


Module.register("MMM-Todoist", {

 defaults: {
	maximumEntries: 10,
	lists: ["inbox"],
	interval: 60,
	fade: true,
	fadePoint: 0.25
 },

maybe a result of the fork from Wunderlist.

you can see that with pm2 logs

0|mm       | Create new todo fetcher for list: inbox - Interval: 60000
the added value in the config.js doesn’t accept.

so I write the correct list in MMM-Todoist.js

 defaults: {
	maximumEntries: 10,
	lists: ["198511455"],
	interval: 60,
	fade: true,
	fadePoint: 0.25
save and restart MM with pm2 restart mm

then look at pm2 logs

|mm       | Create new todo fetcher for list: 198511455 - Interval: 60000
tadaaaa ;-) the correct fetcher ^^

but why the config.js variable not in use — I don’t know ;-)

**Comment by Ooozaboo** [comment]
I have the Same problem, currently displays nothing, It will load the title, but nothing below that.
It seems to load OK in the logs: 0|mm       | Create new todo fetcher for list: inbox - Interval: 3000
Ive tried it with a different list number by editing it in the MMM-Todoist.js, that didnt work for me.
I've rolled back to COMMIT 4a91154, that didnt work.
I was using Wunderlist before Todoist on my magic mirror, not sure if that causes any issues, but any suggestions would be appreciated as I'm stuck now.

**Comment by Sunburner** [comment]
I am in the exact same boat as Ooozaboo.  Was using Wunderlist, switching to todoist for the Alexa integration.  Now i can get the title on the mirror but no entries. 

Being on this boat stinks.

**Comment by tkotz8105** [comment]
I wrote a fix a month ago or so to the ‘no due date tasks’ issue so they will now sort to the end. I think Chris Booker hasn’t had time to implement but the fix is available at my MMM-Todoist fork at: https://github.com/tkotz8105/MMM-Todoist

**Comment by Ooozaboo** [comment]
Nice one tkotz8105! I've installed your fork and it's working! Here's what I did to get it going. 1) Renamed the todoist folder in modules to MMM-Todoistold as git clone moaned that the folder already existed. 2) ran git clone https://github.com/tkotz8105/MMM-Todoist.git 3) npm install from the new todist module folder. 4) updated the config.js file,  changing the list number to one of my todoits lists. 
Sunburner, let us know how you get on.
